cipher/src/block.rs

```diff
@@ -250,9 +250,9 @@ pub trait BlockDecrypt: BlockSizeUser {
     #[cfg(feature = "block-padding")]
     #[cfg_attr(docsrs, doc(cfg(feature = "block-padding")))]
     #[inline]
-    fn decrypt_padded_inout<'inp, 'out, P: Padding<Self::BlockSize>>(
+    fn decrypt_padded_inout<'out, P: Padding<Self::BlockSize>>(
```

Comment:
Technically, removing lifetime argument is a breaking change.

Reply:
Okay, do I need to take any action for this?